.github/pull_request_template.md

```diff
@@ -1,6 +1,14 @@
 ## Description
 
 <!-- prefix the PR title with feat: to increment the minor version, or fix: to increment the patch version -->
-<!-- Ensure all code can be tested offline with unmarked tests. Online tests can be marked as 'integration' -->
 
 <!--What are the changes in this PR? What is the effect? etc.-->
+
```

Comment:
<!-- Update documentation if any changes are made to existing features or APIs -->

Reply:
```suggestion
<!-- Update documentation if any changes are made to existing features or APIs -->

```